controllers/userStatus.js

```diff
@@ -143,4 +143,41 @@ const updateAllUserStatus = async (req, res) => {
   }
 };
 
-module.exports = { deleteUserStatus, getUserStatus, getAllUserStatus, updateUserStatus, updateAllUserStatus };
+/**
+ * Retrieve the idle users based on their task status - in progress , assigned
```

Comment:
```suggestion
 * Retrieve the idle users based on their task status - in progress, assigned
```

Reply:
Idle users based on task status which is either assigned or in progress. 
Did not understand this.
How are these users idle then?

Reply:
As of now users can change their status to idle active or OOO from my site. moving forward this will be decided from their task if they have a task assigned on their name or a task in progress they are active else they would be considered idle.

Reply:
Done